docs/sql-ref-syntax-qry-select-tvf.md

```diff
@@ -0,0 +1,131 @@
+---
+layout: global
+title: Table-valued Functions (TVF)
+displayTitle: Table-valued Functions (TVF)
+license: |
+  Licensed to the Apache Software Foundation (ASF) under one or more
+  contributor license agreements.  See the NOTICE file distributed with
+  this work for additional information regarding copyright ownership.
+  The ASF licenses this file to You under the Apache License, Version 2.0
+  (the "License"); you may not use this file except in compliance with
+  the License.  You may obtain a copy of the License at
+ 
+     http://www.apache.org/licenses/LICENSE-2.0
+ 
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+---
+
+### Description
+
+A table-valued function (TVF) is a function that returns a relation or a set of rows.
```

Comment:
In Spark SQL, we have two types of built-in table-valued functions. 1) Can be specified in the FROM clause. For example `range`; 2) Can be used in SELECT. For example, `explode` and so on.  

@huaxingao Here, we only have the first type. Could you add the second?